### PR TITLE
Some more dartdoc

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
I thought I had more time to write docs, but this is what I got written before I got pulled away.

/cc @Hixie

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -998,6 +998,7 @@ class BlockBody extends MultiChildRenderObjectWidget {
   }
 }
 
+/// A base class for widgets that accept [Positioned] children.
 abstract class StackRenderObjectWidgetBase extends MultiChildRenderObjectWidget {
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/basic.dart]: man, this is really not much of a class, huh.

File: packages/flutter/lib/src/widgets/pageable_list.dart [hunk 2]
```diff
@@ -46,17 +50,34 @@ class PageableList extends Scrollable {
...
+  /// The animation curve to use when animating to a given page.
   final Curve curve;
+
+  /// The list of pages themselves.
   final Iterable<Widget> children;
 
   PageableListState createState() => new PageableListState();
 }
 
+/// State for a [PageableList] widgets.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/pageable_list.dart]: grammar

**Comment by Hixie** [comment]
lgtm

**Pull request merged**